.github/workflows/protobuf.yml

```diff
@@ -19,5 +19,11 @@ jobs:
           js: true
           rust: true
           golang: true
+      - name: Cache `.bin` folder
+        id: cache-setup
+        uses: actions/cache@v2
+        with:
+          path: .bin
+          key: ${{ runner.os }}-bin
```

Comment:
This seems insufficient, as the key won't be invalidated when the `dev/proto-generate.sh` file changes (as one example).